app/models/job.rb

```diff
@@ -208,4 +208,12 @@ def set
     @set = SetClient::Set.find(set_uuid).first
   end
 
+  def work_plan
```

Comment:
Helpful method for a job, maybe we should call it say in the lims_data method instead of work_order.work_plan

Reply:
It needs to be called this to get the work plan to show up on a job 😕